README.md

```diff
@@ -19,6 +19,7 @@ Most importantly, we're utilising GDD (Gif Driven Development).
 ### Dev
 
 * Install [NodeJS](https://nodejs.org/en/)
+* Install [npm](https://www.npmjs.com)
```

Comment:
Ok. According to [npm's docs](https://docs.npmjs.com/getting-started/installing-node) it does come together but you still have to update npm. I already had node on my laptop since a long time (my RB laptop only arrives tomorrow), so maybe something changed or maybe something was wrong my setup.